vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteSpaceController.java

```diff
@@ -25,8 +26,8 @@
     public ResponseEntity<?> deleteSpace(@PathVariable String id) {
         try {
             spaceManager.deleteSpaceById(id);
-        } catch (SpaceDoesNotExistException spaceDoesNotExistException) {
-            logger.error("Could not delete space.", spaceDoesNotExistException);
+        } catch (SpaceDoesNotExistException | SpaceLinkDoesNotExistException exception) {
```

Comment:
Isn't SpaceLinkDoesNotExistException thrown when there is no spacelink that references the space to be deleted? Why would the server return a bad request? It would be absolutely fine to delete a space that isn't linked from anywhere.